component/core/src/main/java/org/exoplatform/social/core/jpa/storage/RDBMSIdentityStorageImpl.java

```diff
@@ -683,7 +683,11 @@ public void processEnabledIdentity(Identity identity, boolean isEnable) {
       identity = profileFilter.getViewerIdentity();
     }
     if (OrganizationIdentityProvider.NAME.equals(providerId)) {
-      return getProfileSearchConnector().search(identity, profileFilter, type, offset, limit);
+      List<Identity> identities = getProfileSearchConnector().search(identity, profileFilter, type, offset, limit);
```

Comment:
IMO you should use forceLoadOrReloadProfile param to not force the profile loading in all cases.

Reply:
we should not query each identity from database, this will cause performance regression. Are firstname/lastname stored in ES without accents ?

Reply:
No in ES nothing is stored with accents.